### DISCUSSION TITLE
¿Se pueden usar estructuras que no se hayan visto en clases?

### DISCUSSION THREAD

**Discussion by Utmite** [IIC2133-PUC/2024-2#102] [Tarea 1] [answered]
Hola, buenas.
Me pregunto si se pueden usar estructuras como median-heap o hash table para la tarea.
Muchas gracias por su atención.

**Comment by tomastrivino** [comment, marked as answer]
Hola,
si bien existen diversas estructuras que puedes utilizar, **la tarea se puede hacer con las que se han visto en clases hasta el momento**. Asegúrate de cumplir las complejidades solicitadas y de explicar tu implementación en el informe.

Saludos
